base/libgit2/reference.jl

```diff
@@ -30,24 +30,49 @@ function isorphan(repo::GitRepo)
     r != 0
 end
 
+"""
+    LibGit2.head(repo::GitRepo) -> GitReference
+
+Returns a `GitReference` to the current HEAD of `repo`.
+"""
 function head(repo::GitRepo)
     head_ptr_ptr = Ref{Ptr{Void}}(C_NULL)
     @check ccall((:git_repository_head, :libgit2), Cint,
                   (Ptr{Ptr{Void}}, Ptr{Void}), head_ptr_ptr, repo.ptr)
     return GitReference(repo, head_ptr_ptr[])
 end
 
+"""
+    LibGit2.shortname(ref::GitReference)
+
+Returns a shortened version of the name of `ref` that's
+"human-readable".
```

Comment:
in what way? is there an xref to the libgit2 or git-git docs that explains what they mean by this?